example/package.json

```diff
@@ -1,29 +1,8 @@
 {
```

Comment:
Most, if not all, of these deps are handled in the root of the project (or loaded externally), so need for them here.